.github/workflows/update-version.yml

```diff
@@ -30,6 +30,12 @@ jobs:
           poetry update
           git add pyproject.toml poetry.lock
           git commit -m 'kmir/{pyproject.toml,poetry.lock}: sync Poetry files' || true
+      - name: 'Update K release tag'
+        run: |
+          PATH="/home/github-runner/.local/bin:$PATH"
```

Comment:
Why is setting `PATH` necessary here?

Reply:
Ah, I missed `echo ${HOME}/.local/bin >> $GITHUB_PATH` above. Will remove this line